src/routes/dashboard/project/[id]/settings/Users.svelte

```diff
@@ -180,29 +199,44 @@
 						/>
 					</TableBodyCell>
 					<TableBodyCell>
-						<Select id="invite_is_admin" bind:value={invite_role} required>
-							<option value="owner">Owner</option>
-							<option value="admin">Admin</option>
-							<option value="user">User</option>
+						<Select class="min-w-24" id="invite_is_admin" bind:value={invite_role} required>
+							{#each canEditAdminLevels as adminLevel}
```

Comment:
```suggestion
							{#each can_edit_admin_levels as admin_level}
```
Ca me fait bizzare dcp l'alternance entre camelCase et snake_case